ote/src/clj/ote/services/transport.clj

```diff
@@ -583,7 +663,19 @@
              :params
              user :user
              form-data :body}
-        (http/transit-response (http/transit-request form-data))) ;THIS IS JUST PLACEHOLDER
+            (authorization/with-transport-operator-check
+              db user (Long/parseLong operator-id)
+              #(manage-adding-users-to-operator email db user (Long/parseLong operator-id)
+                                                (http/transit-request form-data))))
+
+      ;; This is not ready, only implemented to help add-member implementation
```

Comment:
Proposal
```
DELETE /transport-operator/:operator-id/users/?email=myemail@address.com
```
(Although the resource could be `/transport-operator/:operator-id/member/?` but "users" is already used in this domain so...)

FYI route's`params` key contains url query strings as a map ({"email" "myemail@address.com"})
For an example see GET /admin/member?userid